src/prefect/cli/flow_run.py

```diff
@@ -139,13 +142,33 @@ async def cancel(id: UUID):
 
 
 @flow_run_app.command()
-async def logs(id: UUID):
+async def logs(
+    id: UUID,
+    head: bool = typer.Option(
+        False,
+        "--head",
+        "-h",
+        help=f"Show the first {LOGS_HEAD_DEFAULT_NUM_LINES} lines of logs instead of the last 10 lines",
+    ),
+    num_lines: int = typer.Option(
+        LOGS_HEAD_DEFAULT_NUM_LINES,
+        "--num-lines",
+        "-n",
+        help=f"Number of lines to show when using the --head flag",
+        min=1,
+    ),
+):
     """
     View logs for a flow run.
     """
-    page_size = 200
+    # Pagination - API returns max 200 lines at a time
```

Comment:
Maybe make a note that this is what `LOGS_PAGE_SIZE` refers to?

Reply:
Good idea